### PR TITLE
Antennas Geolocalization

### PR DESCRIPTION
**Pull request by Zeeerooo**
- Created method to geolocalized antennas, and update them to the database
- Added job to the schedduler for antennas geolocalization
- Refactor code

### CODE REVIEW

File: config.py [hunk 1]
```diff
@@ -28,3 +28,10 @@ class AdminUser:
     login = "this-really-needs-to-be-changed"
     email = "this-really-needs-to-be-changed"
     password = "this-really-needs-to-be-changed"
+
+
+class OpenCellIdToken:
+    """
+    Open CellId Key token used in the antenna geolocalization process
+    """
+    token = "this-really-needs-to-be-changed"
```
Comment (camilog) [inline config.py]: what?

**Comment by camilog** [comment]
LGTM

File: config.py (hunk 1 above)
Comment (sebablasko) [inline config.py]: maybe, it should be in a config.xml file

Comment (sebablasko) [inline config.py]: the newline!

Comment (Zeeerooo) [inline config.py]: I'll make the change to a .xml in a new pull request. The new line is comming!

**Comment by sebablasko** [comment]
LGTM

**Pull request merged**